src/ui/RunEditor.tsx

```diff
@@ -342,17 +354,29 @@ export class RunEditor extends React.Component<Props, State> {
                     </ContextMenuTrigger>
                 </button>
                 <ContextMenu id="other-button-context-menu">
-                    <MenuItem onClick={(_) => this.clearHistory()}>
-                        Clear History
+                    <MenuItem className="tooltip" onClick={(_) => this.clearHistory()}>
+                        Clear Only History
+                        <span className="tooltip-text">
+                            Splits store the entire history of all runs, including every split time. This information is used by various components. You can clear the history with this. The personal best, the best segment times, and the comparisons will not be affected.
+                        </span>
                     </MenuItem>
-                    <MenuItem onClick={(_) => this.clearTimes()}>
-                        Clear Times
+                    <MenuItem className="tooltip" onClick={(_) => this.clearTimes()}>
+                        Clear All Times
+                        <span className="tooltip-text">
+                            This removes all the times from the splits, including all the history, such that they are completely empty, as if they were just created.
+                        </span>
                     </MenuItem>
-                    <MenuItem onClick={(_) => this.cleanSumOfBest()}>
+                    <MenuItem className="tooltip" onClick={(_) => this.cleanSumOfBest()}>
                         Clean Sum of Best
+                        <span className="tooltip-text">
+                            Allows you to interactively remove potential issues in the segment history that lead to an inaccurate Sum of Best. If you skip a split, whenever you will do the next split, the combined segment time might be faster than the sum of the individual best segments. This will point out all of these and allows you to delete them individually if any of them seem wrong.
```

Comment:
```suggestion
                            Allows you to interactively remove potential issues in the segment history that lead to an inaccurate Sum of Best. If you skip a split, whenever you will do the next split, the combined segment time might be faster than the sum of the individual best segments. This will point out all such occurrences and allow you to delete them individually if any of them seem wrong.
```